src/js/shariff.js

```diff
@@ -257,9 +257,12 @@ module.exports = Shariff;
 // export Shariff class to global (for non-Node users)
 global.Shariff = Shariff;
 
-// initialize .shariff elements
-$('.shariff').each(function() {
-    if (!this.hasOwnProperty('shariff')) {
-        this.shariff = new Shariff(this);
-    }
+$(global.document).ready(function () {
```

Comment:
This syntax for the ready function is deprecated in the newer versions of jquery.
One shall use: `$(function() {});` only.